test/test_clients_api.js

```diff
@@ -24,7 +24,7 @@ let token;
 describe('Clients API', () => {
   before((done) => {
     const pass = btoa('admin:' + config.get('adminPass'));
-    server.post(endpointPrefix + '/users/auth')
+    server.post(endpointPrefix + '/auth/basic')
```

Comment:
this "before" could use the common file now, but maybe later in a separate patch